### ISSUE TITLE
"This monitor is not controllable" after G-sync state change

### ISSUE THREAD

**Issue by Devocub** [emoacht/Monitorian#60]
Win10 x64 latest, monitorian 2.1.0

Happens always after g-sync state change. It is controllable! (always)
![image](https://user-images.githubusercontent.com/2735824/69028064-e4415580-09e1-11ea-9f68-589055e8c993.png)

Restarted the program
![image](https://user-images.githubusercontent.com/2735824/69028104-fde29d00-09e1-11ea-9f7b-4fad18a853c0.png)

Also i tested pre 2.2.0. Same. If i disable or enable G-sync then it show screen in list for few seconds, after shows red "not controllable" for few seconds and disappear. After program restart everything is ok. (Also program doesn't respond to click (in tray) for 3-5 seconds after start)

**Comment by emoacht** [comment]
Thank you for your report.
Could you provide more detailed information? Otherwise, it is hard to understand what is happening.
At least I need the probe.log before and after the "G-Sync" state changes.

> (Also program doesn't respond to click (in tray) for 3-5 seconds after start)

It's normal. Menu will not be ready to open until the first scan completes.

**Comment by Devocub** [comment]
probe is same before and after except this
![image](https://user-images.githubusercontent.com/2735824/69168066-07b1f080-0b07-11ea-9970-d5c6b8bd9f97.png)

https://www.dropbox.com/s/xmz060td2klehyd/video%20of%20bug.mp4?dl=1

**Comment by emoacht** [comment]
I am sorry to say but if you are not willing to provide meaningful information, I can do nothing.

**Comment by Devocub** [comment]
oh, I thought it should work if it's same
[probe, no g-sync.log](https://github.com/emoacht/Monitorian/files/3865467/probe.no.g-sync.log)

[probe after enabling g-sync (not controllable).log](https://github.com/emoacht/Monitorian/files/3865468/probe.after.enabling.g-sync.not.controllable.log)

**Comment by emoacht** [comment]
Thanks for probe.logs.
A possible cause is that the monitor taken before the change is no longer valid after the change. It means the monitor is treated as a different one internally.
Since it may have impact on the design of data model of monitor as immutable object, I need to make some design decisions.

**Comment by emoacht** [comment]
This issue may be changed in Ver 2.3.0.

**Comment by Devocub** [comment]
Seem to work nice!

**Comment by Wasifatehi** [comment]
Hey, my PC had a driver update from Intel related to Display, after that it's showing that "this monitor isn't controllable". So as It was set at lowe brightness the level stayed the same. Now I can't do anything. I tried to install the latest version of monitorian, it isn't installing. But I've managed to install an older version that is installed but it still saying the same. I did roll back to the previous driver update but isn't working at all.
